.env

```diff
@@ -0,0 +1,18 @@
+ADMINS="Local Admin=admin@example.com"
+BROWSERS=chrome,firefox
+CACHE_URL=redis://cache:6379/0
+DATABASE_URL=postgres://postgres@postgres:5432/db
+DEBUG='True'
+DJANGO_LIVE_TEST_SERVER_ADDRESS='web'
+EMAIL_URL=smtp://mail:1025
+GROUPS_EMAIL=groups@example.com
+HOSTS=web,localhost,0.0.0.0
+HTTPS='False'
+LEAN_MANAGERS="Local Manager=leanmanager@example.com"
+PYTEST_ADDOPTS=--driver=Remote
+PYTHONDONTWRITEBYTECODE='True'
+SECRET_KEY=development_secret_key
+SELENIUM_HOST=webdriver
+DJANGO_ENV=local
+ALGOLIA_API_KEY=d1c9139b2271e35f44a29b12dddb4b06
+DEFAULT_STORAGE_DSN=s3://AKIATGMPLQZUBUOMZKFB:WVFQpgc6TznKHtl%2FjvqtrkCjeimpkua9ea%2Bf4i%2B7@eahub-test-30dbe6e2fb3448ccacfc2e56e944-3f4cc57.divio-media.org.s3.amazonaws.com/?auth=s3v4&domain=eahub-test-30dbe6e2fb3448ccacfc2e56e944-3f4cc57.divio-media.org
```

Comment:
I'm aware of those being in git, FYI that's just test credentials.

Reply:
What I need to do with it?